### PR TITLE
Template Messages

### PR DESCRIPTION
**Pull request by Raemon**
Makes the "Message" button on the SunshineNewUsersItem use a default template, which can later be edited.

I also ended up tweaking the styling of some buttons as well

image

### CODE REVIEW

File: packages/lesswrong/components/messaging/ConversationPage.tsx [hunk 1]
```diff
@@ -59,7 +68,7 @@ const ConversationPage = ({ documentId, terms, currentUser, classes }: {
     </div>
   }
 
-  if (loading) return <Loading />
+  if (loading || loadingTemplate) return <Loading />
```
Comment (Raemon) [inline packages/lesswrong/components/messaging/ConversationPage.tsx:71]: This probably adds a very slight delay to all _non_ templated messages, which is barely noticeable but is worth checking "can we not do that?"

Comment (Discordius) [reply packages/lesswrong/components/messaging/ConversationPage.tsx:71]: Yeah, you can just check for `&& query?.templateCommentId`

File: packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.tsx [hunk 2]
```diff
@@ -167,6 +171,8 @@ const SunshineNewUsersItem = ({ user, classes, updateUser, allowContentPreview=t
   const hiddenPostCount = user.maxPostCount - user.postCount
   const hiddenCommentCount = user.maxCommentCount - user.commentCount
 
+  const templateId = defaultModeratorComments.get()[0]
+  console.log(defaultModeratorComments.get(), templateId)
```
Comment (Discordius) [inline packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.tsx]: Spurious console log

File: packages/lesswrong/components/messaging/NewConversationButton.tsx [hunk 3]
```diff
@@ -23,7 +25,8 @@ const NewConversationButton = ({ user, currentUser, children }: {
       data: {participantIds:[user._id, currentUser._id], ...alignmentFields},
     })
     const conversationId = response.data.createConversation.data._id
-    history.push({pathname: `/inbox/${conversationId}`})
+    const search = templateCommentId ? {search:`?${qs.stringify({templateCommentId: templateCommentId})}`} : {}
```
Comment (Discordius) [inline packages/lesswrong/components/messaging/NewConversationButton.tsx:28]: Double check that this doesn't add `?templateCommentId=undefined` or something like that every time someone clicks the message button on someone's profile

**Review by Discordius — CHANGES_REQUESTED** [review]

File: packages/lesswrong/components/messaging/NewConversationButton.tsx (hunk 3 above)
Comment (Raemon) [reply packages/lesswrong/components/messaging/NewConversationButton.tsx:28]: Oh yeah I did check that last time. It does not.

**Review by Discordius — APPROVED** [review]

**Pull request merged**